doc/src/sphinx/develop/setup_dev.rst

```diff
@@ -23,14 +25,17 @@ The repository `clowder` containes the web frontend and is required. It should b
 
 ::
 
-  > git clone https://opensource.ncsa.illinois.edu/bitbucket/scm/cats/clowder.git
+  $ git clone https://github.com/clowder-framework/clowder
+  Depricated: https://opensource.ncsa.illinois.edu/bitbucket/scm/cats/clowder.git
```

Comment:
just remove the link

Reply:
Per discussion on Slack with Luigi, we will remove this page and move the remaining useful bits into the "Customizing" page.